dev-docs/publisher-api-reference/aliasBidder.md

```diff
@@ -28,6 +28,7 @@ The options object supports these parameters:
 | Option Parameter    | Type    | Description             |
 |------------|---------|---------------------------------|
 | gvlid | integer | IAB Global Vendor List ID for this alias for use with the [TCF control module](/dev-docs/modules/tcfControl.html). |
+| useBaseGvlid | boolean | Flag determining if the GVL ID of the original adapter should be re-used |
```

Comment:
@mkomorski could you add a `scope` column, which indicates which parameters are required or optional? I guess @ChrisHuie went through this with you yesterday :blush: , just wanted to put it here for the record.

Thanks to @lpagnypubstack for pointing this out :pray: